src/engraving/libmscore/chord.cpp

```diff
@@ -1365,6 +1365,7 @@ qreal Chord::defaultStemLength() const
     }
 
     qreal normalStemLen = isSmall() ? 2.5 : 3.5;
+    /* We no longer need to adjust stem hight for dot-hook collisions (Issue #9095)
```

Comment:
Better use #if 0... #endif. Here you're creating a compiler warning "comment inside comment"

Reply:
Better to remove this code completely